apps/acrs/index.js

```diff
@@ -622,8 +623,8 @@ module.exports = {
       next: '/decision-postal-address'
     },
     '/email-decision': {
-      behaviours: SaveFormSession,
-      fields: ['is-decision-by-email', 'is-decision-by-email-detail'],
+      behaviours: [SaveFormSession, SaveDecisionEmail],
+      fields: ['is-decision-by-email', 'is-decision-by-email-detail', 'decision-email'],
```

Comment:
Why is the `decision-email` field needed given that it is visually hidden in `/fields/index.js`?

Reply:
removed not needed